idea/gearexe/frontend/src/components/ui/syntax-highlighter/syntax-highlighter.tsx

```diff
@@ -0,0 +1,36 @@
+import ReactHighlighter, { SyntaxHighlighterProps } from 'react-syntax-highlighter';
+import { stackoverflowDark } from 'react-syntax-highlighter/dist/esm/styles/hljs';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const Highlighter = ReactHighlighter as any as React.FC<SyntaxHighlighterProps>;
```

Comment:
```suggestion
const Highlighter = ReactHighlighter as unknown as React.FC<SyntaxHighlighterProps>;
```
Wouldn't be better like that?